pontoon/administration/management/commands/import_verbatim.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+
+import requests
+
+from django.contrib.auth.models import User
+from django.core.management.base import (
+    BaseCommand,
+)
+from pontoon.base.models import Translation, Locale
+from django.utils import timezone
+
+
+class Command(BaseCommand):
+    help = 'Import translation authors and dates from Verbatim.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--project',
+            action='store',
+            dest='project',
+            required=True,
+            help='Project slug to import Verbatim data to'
+        )
+
+        parser.add_argument(
+            '--locale',
+            action='store',
+            dest='locale',
+            required=True,
+            help='Locale code to import Verbatim data to'
+        )
+
+        parser.add_argument(
+            '--filename',
+            action='store',
+            dest='filename',
+            required=True,
+            help='File name to import Verbatim data from'
+        )
+
+    def handle(self, *args, **options):
+        url = 'http://svn.mozilla.org/projects/l10n-misc/trunk/pontoon/verbatim/' + options['filename']
+        verbatim = requests.get(url).json()
+
+        locale_translators_map = dict(Locale.objects.values_list('code', 'translators_group'))
+
+        for path in verbatim.keys():
+            for source in verbatim[path]:
+                translations = Translation.objects.filter(
+                    entity__resource__project__slug=options['project'],
+                    entity__resource__path=path,
+                    entity__string=source,
+                    locale__code=options['locale'],
+                    user=None,
+                    string=verbatim[path][source]["translation"]
+                )
+                for t in translations:
+                    try:
+                        # Save user
+                        email = verbatim[path][source]["author"]
+                        user = User.objects.get(email=email)
+                        t.user = user
+                        t.approved_user = user
+
+                        # Save date
+                        date = verbatim[path][source]["date"].split("+")[0]
+                        dateObj = timezone.make_aware(datetime.strptime(date, "%Y-%m-%d %H:%M:%S"))
+                        t.date = dateObj
+                        t.approved_date = dateObj
+
+                        t.save()
+
+                        user.groups.add(locale_translators_map[options['locale']])
+                    except User.DoesNotExist:
+                        # TODO: create missing user?
+                        self.stdout.write("Pontoon user not found: " + email)
```

Comment:
This file was replaced with pontoon/sync/management/commands/import_pootle.py, feel free to remove it.